.github/workflows/publish-operatorhub.yml

```diff
@@ -1,15 +1,13 @@
 name: Publish to OperatorHub
 
 on:
+  release:
+    types: [published]
   workflow_dispatch:
     inputs:
-      version:
-        description: 'Operator version to publish (e.g., 0.88.0)'
-        required: false
-        type: string
       tag:
-        description: 'Existing release tag to publish (e.g., v0.88.0)'
-        required: false
+        description: 'Release tag to publish (e.g., v0.88.0)'
+        required: true
```

Comment:
No. It's not doing what is says it's doing. With this it will break the workflow for manual runs. Is this human or AI ?